services/resource-usage-tracker/src/simcore_service_resource_usage_tracker/resource_tracker_cli_placeholder.py

```diff
@@ -0,0 +1,135 @@
+import datetime
+import json
+import logging
+from typing import Any
+
+import arrow
+from fastapi import FastAPI
+from models_library.users import UserID
+from prometheus_api_client import PrometheusConnect
+from simcore_service_resource_usage_tracker.modules.prometheus import (
+    get_prometheus_api_client,
+)
+
+# This script assumes everywhere that the minimum granularity of the data is 1 minute
+# Setting it smaller is unreasonable at least for prometheus, as the scraping interval is apprx. eq. to 1 h
+
+
+_logger = logging.getLogger(__name__)
+
+
+def _assure_dict_entry_exists(
+    metric_data, max_values_per_docker_id, image, userid
+) -> None:
+    for metric in metric_data:
+        current_id = metric["metric"]["id"]
+        if current_id not in max_values_per_docker_id.keys():
+            max_values_per_docker_id[current_id] = {
+                "container_uuid": metric["metric"]["container_label_uuid"],
+                "cpu_seconds": 0,
+                "uptime_minutes": 0,
+                "nano_cpu_limits": 0,
+                "egress_bytes": 0,
+                "image": image,
+                "user_id": userid,
+            }
+
+
+async def _evaluate_service_resource_usage(
+    prometheus_client: PrometheusConnect,
+    start_time: datetime.datetime,
+    stop_time: datetime.datetime,
+    user_id: UserID,
+    uuid: str = ".*",
+    image: str = "registry.osparc.io/simcore/services/dynamic/jupyter-smash:3.0.9",
+) -> dict[str, Any]:
+    max_values_per_docker_id: dict[str, Any] = {}
+    time_delta = stop_time - start_time
+    minutes = round(time_delta.total_seconds() / 60)
+
+    for current_datetime in [
+        stop_time - datetime.timedelta(minutes=i) for i in range(minutes)
+    ]:
+        rfc3339_str = current_datetime.isoformat("T")
```

Comment:
Does this take into consideration UTC? Also is it better to use arrow for these operations?

Reply:
This is a placeholder with the old script provided by DevOps, this is not my code and will be removed in the upcoming PRs. But currently unit tests already cover this situation so i want to leave it there for now :+1: